tests/runner-templates/build-test-macos

```diff
@@ -28,7 +28,7 @@ jobs:
     steps:
     - name: Cancel previous runs on the same branch
       if: ${{ github.ref != 'refs/heads/main' }}
-      uses: styfle/cancel-workflow-action@0.9.0
+      uses: styfle/cancel-workflow-action@0.9.1
```

Comment:
right. in this case we updated the yml files, so when you rebuild the workflow files, it regresses.
I don't know what the ideal solution is. My understanding is that it all boils down to us wanting many. fine-grained, workflow files in order to allow re-running few tests, instead of all tests.